macstadium-orka-server/src/main/java/com/macstadium/orka/client/OrkaClient.java

```diff
@@ -171,6 +172,15 @@ private String executeCall(Request request) throws IOException {
 
     @Override
     public void close() throws IOException {
-        this.delete(this.endpoint + TOKEN_PATH, "");
+        String response = this.get(this.endpoint + HEALTH_PATH);
+        Gson gson = new Gson();
+        JsonObject jsonObject = gson.fromJson(response, JsonObject.class);
+        String apiVersionString = jsonObject.get("api_version").toString();
+        apiVersionString.replace(".", "");
```

Comment:
This does nothing. String are immutable and `replace` generates a new string. So we still pass `2.2.0` to `parseInt`.
You would either need to overwrite the variable, create a new variable or directly call `int apiVersion = Integer.parseInt(apiVersionString.replace(".", ""));`